nodes/number/float_math.py

```diff
@@ -58,6 +66,9 @@ def create(self):
 
     def draw(self, layout):
         layout.prop(self, "operation", text = "")
+        
+    def draw_label(self):
```

Comment:
The effort to do this is very low. 
max. 5 lines
I can never know wether we will use this more in the future. Also then this function blends better into the others. I rarely use underscores in variable and function names